server/middleware/validationMiddleware.test.ts

```diff
@@ -45,6 +46,12 @@ describe('validationMiddleware', () => {
       outOfBoundFilename: string
     }
 
+    class DummyAddress {
```

Comment:
Does this cover anything which `server/validators/doesNotContainMarkup.test.ts` doesnt?

Reply:
Well, it's a final check on a wider-scoped function - this checks via the `validationMiddleware` whilst the other tests are limited to the `validate()` function.